tools/pepquery/pepquery.xml

```diff
@@ -19,6 +19,11 @@
             #set $db_file = $re.sub('\s','_',$re.sub('[.][^.]*$','',$req_inputs.db_file.display_name.split('/')[-1])) + ".fa"
             ln -s '$req_inputs.spectrum_file' '$spectrum_file' &&
             ln -s '$req_inputs.db_file' '$db_file' &&
+            #if $req_inputs.tags.tagType == 'pFind':
+                pepquery main.java.util.generateTags $req_inputs.tags.qry_res > qry.tags &&
+                ls -l $req_inputs.tags.qry_res qry.tags &&
```

Comment:
is that ls and cat still needed?

Reply:
@bgruening   No.  That was just debugging.  I'll get that removed. Thanks.